docs/snippets/ov_network_state_intro.cpp

```diff
@@ -0,0 +1,240 @@
+    manager.run_passes(ov_model);
+    //! [ov:apply_low_latency_2]
+
+    auto hd_specific_model = core.compile_model(ov_model);
+    // Try to find the Variable by name
+    auto infer_request = hd_specific_model.create_infer_request();
+    auto states = infer_request.query_state();
+    for (auto& state : states) {
+        auto name = state.get_name();
+        if (name == state_name) {
+            // some actions
+        }
+    }
+    //! [ov:low_latency_2]
+
+    //! [ov:low_latency_2_use_parameters]
+    manager.register_pass<ov::pass::LowLatency2>(false);
+    //! [ov:low_latency_2_use_parameters]
+}
+
+void replace_non_reshapable_const() {
+    //! [ov:replace_const]
+    // OpenVINO example. How to replace a Constant with hardcoded values of shapes in the network with another one with the new values.
+    // Assume we know which Constant (const_with_hardcoded_shape) prevents the reshape from being applied.
+    // Then we can find this Constant by name on the network and replace it with a new one with the correct shape.
+    ov::Core core;
+    auto model = core.read_model("path_to_model");
+    // Creating the new Constant with a correct shape.
+    // For the example shown in the picture above, the new values of the Constant should be 1, 1, 10 instead of 1, 49, 10
+    auto new_const = std::make_shared<ov::opset8::Constant>( /*type, shape, value_with_correct_shape*/ );
+    for (const auto& node : model->get_ops()) {
+        // Trying to find the problematic Constant by name.
+        if (node->get_friendly_name() == "name_of_non_reshapable_const") {
+            auto const_with_hardcoded_shape = std::dynamic_pointer_cast<ov::opset8::Constant>(node);
+            // Replacing the problematic Constant with a new one. Do this for all the problematic Constants in the network, then
+            // you can apply the reshape feature.
+            ov::replace_node(const_with_hardcoded_shape, new_const);
+        }
+    }
+    //! [ov:replace_const]
+}
+
+void apply_make_stateful_tensor_names() {
+    //! [ov:make_stateful_tensor_names]
+    ov::Core core;
+    auto ov_model = core.read_model("path_to_the_model");
+    std::map<std::string, std::string> tensor_names = {{"tensor_name_1", "tensor_name_4"},
+                                                  {"tensor_name_3", "tensor_name_6"}};
+    ov::pass::Manager manager;
+    manager.register_pass<ov::pass::MakeStateful>(tensor_names);
+    manager.run_passes(ov_model);
+    //! [ov:make_stateful_tensor_names]
+}
+
+void apply_make_stateful_ov_nodes() {
+    //! [ov:make_stateful_ov_nodes]
+    ov::Core core;
+    auto ov_model = core.read_model("path_to_the_model");
+    // Parameter_1, Result_1, Parameter_3, Result_3 are shared_ptr<Parameter/Result> in the ov_model
+    std::vector<std::pair<std::shared_ptr<ov::opset8::Parameter>, std::shared_ptr<ov::opset8::Result>>> pairs
+            = {/*Parameter_1, Result_1, Parameter_3, Result_3*/};
```

Comment:
Shouldn't it be like
```suggestion
            = {/*std::make_pair(Parameter_1, Result_1), std::make_pair(Parameter_3, Result_3)*/};
```